.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,42 @@
+# Code of Conduct
```

Comment:
is this a template from somewhere? I'm not super opposed, but I think I'd like to review any new code or pledge separately from a CI/CD change.

Reply:
Some of my code is from Faust, sorry about that. I can remove this.

Reply:
100% agree. While I'm also not super opposed, in general on small projects I've noticed the culture invariably becomes whatever way the couple of maintainers behave regardless of any documented statements. So for now this would just add noise to me. 

This is very different than larger teams/companies where you do need stated expectations... but if we get to that point we can always add later if needed.

And if it is something someone feels strongly on, well, noise isn't horrible... it's just like "why??"